src/igvfd/schemas/biomarker.json

```diff
@@ -0,0 +1,114 @@
+{
+    "title": "Biomarker",
+    "description": "Schema for submitting a biomarker.",
+    "$id": "/profiles/biomarker.json",
+    "$schema": "https://json-schema.org/draft/2020-12/schema",
+    "type": "object",
+    "required": [
+        "name",
+        "quantification"
+    ],
+    "identifyingProperties": [
+        "uuid",
+        "name_quantification"
+    ],
+    "additionalProperties": false,
+    "mixinProperties": [
+        {
+            "$ref": "mixins.json#/basic_item"
```

Comment:
why there is no attribution?

Reply:
I didn't think this is needed, but I have added it.